### ISSUE TITLE
Unexpected error fetching data

### ISSUE THREAD

**Issue by austinmroczek** [craigjmidwinter/total-connect-client#183]
Here a log entry from my HA system.  Not sure if this is related to #180 or not.  This error was received during the system-wide Total Connect outage the other day.

```txt

Logger: homeassistant.components.totalconnect
Source: components/totalconnect/__init__.py:82
Integration: Total Connect (documentation, issues)
First occurred: March 13, 2022, 3:54:36 PM (798 occurrences)
Last logged: March 13, 2022, 10:36:13 PM

Unexpected error fetching totalconnect data: Unknown fault occured
Traceback (most recent call last):
  File "/srv/homeassistant/lib/python3.9/site-packages/homeassistant/helpers/update_coordinator.py", line 187, in _async_refresh
    self.data = await self._async_update_data()
  File "/srv/homeassistant/lib/python3.9/site-packages/homeassistant/components/totalconnect/__init__.py", line 76, in _async_update_data
    await self.hass.async_add_executor_job(self.sync_update_data)
  File "/usr/lib/python3.9/concurrent/futures/thread.py", line 52, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/srv/homeassistant/lib/python3.9/site-packages/homeassistant/components/totalconnect/__init__.py", line 82, in sync_update_data
    self.client.locations[location_id].get_panel_meta_data()
  File "/srv/homeassistant/lib/python3.9/site-packages/total_connect_client/location.py", line 80, in get_panel_meta_data
    result = self.parent.request("GetPanelMetaDataAndFullStatusEx_V2", (
  File "/srv/homeassistant/lib/python3.9/site-packages/total_connect_client/client.py", line 185, in request
    response = self._send_one_request(operation_name, args)
  File "/srv/homeassistant/lib/python3.9/site-packages/total_connect_client/client.py", line 165, in _send_one_request
    return zeep.helpers.serialize_object(operation_proxy(*args))
  File "/srv/homeassistant/lib/python3.9/site-packages/zeep/proxy.py", line 46, in __call__
    return self._proxy._binding.send(
  File "/srv/homeassistant/lib/python3.9/site-packages/zeep/wsdl/bindings/soap.py", line 135, in send
    return self.process_reply(client, operation_obj, response)
  File "/srv/homeassistant/lib/python3.9/site-packages/zeep/wsdl/bindings/soap.py", line 229, in process_reply
    return self.process_error(doc, operation)
  File "/srv/homeassistant/lib/python3.9/site-packages/zeep/wsdl/bindings/soap.py", line 317, in process_error
    raise Fault(
zeep.exceptions.Fault: Unknown fault occured


```

**Comment by dulitz** [comment]
Yes. See https://github.com/mvantellingen/python-zeep/issues/563 .

Catching zeep.exceptions.Fault and raising something more meaningful indicating a server error might better inform people who see the trace in their logs.

**Comment by austinmroczek** [comment]
fixed by #182
